### PR TITLE
Fixes for issues #625 and #626

### PR DESCRIPTION
**Pull request by dwatteau**
Here's my attempt at fixing the issues #625 and #626 I've previously reported.

### CODE REVIEW

File: configure [hunk 1]
```diff
@@ -2,9 +2,47 @@
...
+
+	return res;
+}
+"
+
+check_c11()
+{
+	msg_checking "for C11 (with atomics support)"
+
+	for flag in -std=gnu11 -std=gnu1x -std=c11 ""
```
Comment (mahkoh) [inline configure]: gnu11 should be sufficient

Comment (dwatteau) [inline configure]: I was thinking of potential compilers where gnu11 isn't available, but other C11 options (or none at all) would be enough to build cmus.

I don't think these fallbacks hurt, unless we really only want gnu11 compatibility (I could be fine with that).

Comment (mahkoh) [inline configure]: gnu1x is certainly unnecessary.

Comment (dwatteau) [inline configure]: Here you go.

**Comment by mahkoh** [comment]
Merged manually.

**Pull request closed without merging**